include/proxsuite-nlp/prox-solver.hxx

```diff
@@ -456,7 +456,7 @@ void ProxNLPSolverTpl<Scalar>::innerLoop(Workspace &workspace,
                      dphi0,
                      results.merit,
                      phi_new - phi0,
-                     static_cast<size_t>(results.al_iters + 1)};
+                     static_cast<unsigned long>(results.al_iters + 1)};
```

Comment:
Wait, I think we should maybe change the logger to use `size_t` instead.

Reply:
Ok, I will change it